app/Http/Transformers/PostTransformer.php

```diff
@@ -30,12 +30,15 @@ public function transform(Post $post)
             'signup_id' => $post->signup_id,
             'northstar_id' => $post->northstar_id,
             'media' => [
-
                 'url' => $post->getMediaUrl(),
                 'original_image_url' => $post->url,
                 'caption' => $post->caption,
             ],
             'tags' => $post->tagSlugs(),
+            'reactions' => [
+                'reacted' => $post->reactions->isNotEmpty(),
+                'total' => isset($post->reactions_count) ? $post->reactions_count : 'unavailable',
```

Comment:
Hmm, can we use `null` instead of `"unavailable"` so the datatype is `int|null` rather than `int|string`?

Reply:
@DFurnes yeah I wasn't sure what to do here because I wanted to somehow signify that "we didn't get the total number of reactions for you" vs "there are 0 reactions." I guess I could use `null` for the first case?

Reply:
Yeah yeah, it's a lil' wonky. I think `null` is decent enough for "we didn't get this for you" since zero reactions would show up as an integer `0`.